doc/radosgw/config-ref.rst

```diff
@@ -954,6 +954,28 @@ Keystone Settings
 :Default: None
 
 
+``rgw keystone admin tenant``
+
+:Description: The OpenStack tenant with admin privilege (Service Tenant) when
```

Comment:
It would be nice to mention that it's about the *name* (not *ID*) of the admin tenant.